upgrade-2.x.sh

```diff
@@ -940,12 +1001,20 @@ target_version=$(echo "$target_version" | tr + _)
 if [ -z "$RESINOS_TAG" ]; then
     RESINOS_TAG=${target_version}-${SLUG}
 fi
-image="${RESINOS_REGISTRY}/${RESINOS_REPO}:${RESINOS_TAG}"
-log "Checking for manifest of ${image}"
-if [ "$(image_exists "$RESINOS_REGISTRY" "$RESINOS_REPO" "$RESINOS_TAG")" = "yes" ]; then
-    log "Manifest found, good to go..."
+log "Attempting host OS update using deltas"
+
+image=$(find_delta "${target_version/_/+}")
```

Comment:
It looks like we're normalizing `$target_version` here, but I don't know the reason.  Is it appropriate to add a comment?

If it is being normalized for use by `find_delta()`, should it be moved into that function along with the other normalization being done thee? It looks like this is the only place in the script where we're making this change.

Reply:
This can be cleaned up further to not require the substitution, have pushed up a cleaner version.